src/modules/decentralized-web-node/dtos/verifiable-credential.dto.ts

```diff
@@ -0,0 +1,79 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsDateString,
+  IsObject,
+  IsOptional,
+  IsString,
+  ValidateNested,
+} from 'class-validator';
+import {
+  CredentialSubject,
+  VerifiableCredential,
+} from '@ew-did-registry/credentials-interface';
+import { Type } from 'class-transformer';
+import { CredentialSchemaOrCredentialStatusDto } from './credential-schema-status.dto';
+import { ProofDto } from './proof.dto';
+
+export class VerifiableCredentialDto
```

Comment:
Can this class be removed? It doesn't look like it is referenced anywhere. Then `proof` and `eip712-domain` could be removed as well?